content/brand/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Pulumi Brand
```

Comment:
NIT: I believe this will render to `Pulumi Brand | Pulumi`, which is a little redundant. If we just use Brand, then we `Brand | Pulumi` which is a bit cleaner IMO.